src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0992.xml

```diff
@@ -703,6 +703,10 @@
                   </destinationSprite>
                   <spinnerSelection>82</spinnerSelection>
                 </brick>
+                <brick type="SetRotationStyleBrick">
+                  <commentedOut>false</commentedOut>
+                  <selection>1</selection>
```

Comment:
Please also add example for all possible selection values.

Reply:
Done!